### PR TITLE
Document some bits of Spawner.

### PR DESCRIPTION
**Pull request by i509VCB**

### CODE REVIEW

**Review by modmuss50 — APPROVED** [review]

File: mappings/net/minecraft/world/gen/PillagerSpawner.mapping [hunk 1]
```diff
@@ -1,3 +1,8 @@
 CLASS net/minecraft/class_3769 net/minecraft/world/gen/PillagerSpawner
 	FIELD field_16652 ticksUntilNextSpawn I
-	METHOD method_16575 spawnOneEntity (Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;Ljava/util/Random;Z)Z
+	METHOD method_16575 spawnPillager (Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;Ljava/util/Random;Z)Z
+		ARG 1 world
+		ARG 2 pos
+		ARG 3 random
+		ARG 4 leader
+			COMMENT specifies whether the pillager is the leader of a patrol
```
Comment (liach) [inline mappings/net/minecraft/world/gen/PillagerSpawner.mapping]:
```suggestion
		ARG 4 captain
			COMMENT whether the pillager is the leader of a patrol
```
https://minecraft.gamepedia.com/Raid_captain#Patrols

File: mappings/net/minecraft/world/gen/Spawner.mapping [hunk 2]
```diff
@@ -1,2 +1,13 @@
 CLASS net/minecraft/class_5304 net/minecraft/world/gen/Spawner
+	COMMENT Handles spawning entities in a world.
+	COMMENT
+	COMMENT <p>A spawner is typically used to spawn entities within a special context, such as cats in a village or wandering traders.
 	METHOD method_6445 spawn (Lnet/minecraft/class_3218;ZZ)I
+		COMMENT Spawns entities into a world.
+		COMMENT
+		COMMENT @return the amount of entities that were spawned. 0 if no entities were spawned.
```
Comment (liach) [inline mappings/net/minecraft/world/gen/Spawner.mapping]:
```suggestion
		COMMENT @return the count of entities spawned
```

**Review by liach — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/world/gen/Spawner.mapping (hunk 2 above)
Comment (i509VCB) [reply mappings/net/minecraft/world/gen/Spawner.mapping]: I'd say `the number of entities spawned` then. `count of entities` really doesn't sound right

File: mappings/net/minecraft/world/gen/Spawner.mapping [hunk 3]
```diff
@@ -1,2 +1,13 @@
 CLASS net/minecraft/class_5304 net/minecraft/world/gen/Spawner
+	COMMENT Handles spawning entities in a world.
+	COMMENT
+	COMMENT <p>A spawner is typically used to spawn entities within a special context, such as cats in a village or wandering traders.
 	METHOD method_6445 spawn (Lnet/minecraft/class_3218;ZZ)I
+		COMMENT Spawns entities into a world.
+		COMMENT
+		COMMENT @return the number of entities that were spawned. 0 if no entities were spawned.
```
Comment (liach) [inline mappings/net/minecraft/world/gen/Spawner.mapping]:
```suggestion
		COMMENT @return the number of entities spawned
```
Redundant; we try to keep `@` clauses noun phrases than full sentences.

**Review by liach — APPROVED** [review]

**Review by Gegy — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Comment by liach** [comment]
The rename only happens to a method that is unlikely used by many mods and is low impact. Should be good to go.

**Pull request merged**